apps/src/lab2/levelEditors/levelData/EditMusicLevelData.tsx

```diff
@@ -39,6 +40,10 @@ const EditMusicLevelData: React.FunctionComponent<EditMusicLevelDataProps> = ({
   }, []);
 
   const [levelData, setLevelData] = useState(initialLevelData);
+  // Immediately set a level, if needed, so we can populate its allowed sounds.
+  if (!levelData.library) {
+    levelData.library = RECOMMENDED_LIBRARY;
```

Comment:
Just confirming that this triggers the `useEffect` to load the library? I would think that we need to use a state update like `setLevelData({...levelData, library: RECOMMENDED_LIBRARY})` but maybe not?

Reply:
It does!